### PR TITLE
RHDEVDOCS-6141 Update the table values in the Sizing requirements

### PR DESCRIPTION
**Pull request by likhithj1996**
Version(s):

Gitops: 1.14 and 1.15

Issue:

[RHDEVDOCS-6141](https://issues.redhat.com/browse/RHDEVDOCS-6141)

Link to docs preview:
[Sizing requirements for GitOps](https://85617--ocpdocs-pr.netlify.app/openshift-gitops/latest/installing_gitops/preparing-gitops-install.html#sizing-requirements-for-gitops_preparing-gitops-install)

QE review:
- [x] QE has approved this change.

SME review: cfang@redhat.com
QE review:
Peer review:
Internal Peer review: @Dhruv-Soni11 

Additional information:

### CODE REVIEW

**Comment by Dhruv-Soni11** [comment]
lgtm, thanks @likhithj1996

File: modules/sizing-requirements-for-gitops.adoc [hunk 1]
```diff
@@ -13,13 +13,13 @@ Every time you install the {gitops-title} Operator, the resources on the namespa
...
 |===
 |Workload |CPU requests |CPU limits |Memory requests |Memory limits
-|argocd-application-controller |1 |2 |1024Mi |2048Mi
-|applicationset-controller |1 |2 |512Mi |1024Mi
-|argocd-server |0.125 |0.5 |128Mi |256Mi
-|argocd-repo-server |0.5 |1 |256Mi |1024Mi
-|argocd-redis |0.25 |0.5 |128Mi |256Mi
-|argocd-dex |0.25 |0.5 |128Mi |256Mi
-|HAProxy |0.25 |0.5 |128Mi |256Mi
+|argocd-application-controller |250m |2m |1024Mi |2048Mi
```
Comment (chengfang) [inline modules/sizing-requirements-for-gitops.adoc:16]: There are change of values, such as from 1 cpu to 250m, and in some other lines,  in addition to the format changes. Are these intended, if so where is the source requesting this change?

**Comment by likhithj1996** [comment]
Hi Cheng Fang,

Thanks for the review and for pointing out the changes in the table values. The data used here is based on Varsha's testing efforts.

@varshab1210 , could you please provide additional context or confirm the results to address Cheng Fang's comments?

File: modules/sizing-requirements-for-gitops.adoc (hunk 1 above)
Comment (varshab1210) [reply modules/sizing-requirements-for-gitops.adoc:16]: Hi @chengfang, these values are taken from direct installation (in August) as @Srivaralakshmi and @Dhruv-Soni11 wanted to maintain consistency in this table and not mention values in decimal.

Comment (varshab1210) [inline modules/sizing-requirements-for-gitops.adoc:16]:
```suggestion
|argocd-application-controller |250m |2 |1024Mi |2048Mi
```

File: modules/sizing-requirements-for-gitops.adoc [hunk 2]
```diff
@@ -13,13 +13,13 @@ Every time you install the {gitops-title} Operator, the resources on the namespa
...
+|applicationset-controller |250m |2m |512Mi |1024Mi
```
Comment (varshab1210) [inline modules/sizing-requirements-for-gitops.adoc:17]:
```suggestion
|applicationset-controller |250m |2 |512Mi |1024Mi
```

File: modules/sizing-requirements-for-gitops.adoc [hunk 3]
```diff
@@ -13,13 +13,13 @@ Every time you install the {gitops-title} Operator, the resources on the namespa
...
+|argocd-server |125m |500m |128Mi |256Mi
+|argocd-repo-server |250m |1m |256Mi |1024Mi
```
Comment (varshab1210) [inline modules/sizing-requirements-for-gitops.adoc:19]:
```suggestion
|argocd-repo-server |250m |1 |256Mi |1024Mi
```

**Review by chengfang — APPROVED** [review]

**Comment by varshab1210** [comment]
LGTM

File: modules/sizing-requirements-for-gitops.adoc (hunk 3 above)
Comment (likhithj1996) [reply modules/sizing-requirements-for-gitops.adoc]: Incorporated the changes as requested.

File: modules/sizing-requirements-for-gitops.adoc (hunk 2 above)
Comment (likhithj1996) [reply modules/sizing-requirements-for-gitops.adoc]: Incorporated the changes as requested.

File: modules/sizing-requirements-for-gitops.adoc (hunk 1 above)
Comment (likhithj1996) [reply modules/sizing-requirements-for-gitops.adoc]: Incorporated the changes as requested.

Comment (likhithj1996) [reply modules/sizing-requirements-for-gitops.adoc]: Incorporated the changes as requested.

**Review by skopacz1 — APPROVED** [review]
Changes LGTM!

**Comment by skopacz1** [comment]
Friendly reminder that if this PR is QE approved, please check the box in the PR description to make it easier to tell for the merge reviewer

**Pull request merged**